RELEASE_NOTES.md

```diff
@@ -1,9 +1,9 @@
 Release notes
 =============
 
-### Next
```

Comment:
Oh, and you should leave the existing release notes in place, and the heading "next". Then you do a separate PR to release the lot, if I remember correctly.

Reply:
Wot George said ^